packages/react-native-jigsaw/package.json

```diff
@@ -25,7 +26,11 @@
     "example": "yarn --cwd example",
     "pods": "cd example && pod-install --quiet",
     "bootstrap": "yarn example && yarn && yarn pods",
-    "postinstall": "expo-yarn-workspaces check-workspace-dependencies"
+    "postinstall": "expo-yarn-workspaces check-workspace-dependencies",
+    "set-up-hooks": "mkdir -p ../.git/hooks && cp -v ../../hooks/* ../../.git/hooks",
```

Comment:
can we just use husky here?  Easier maintenance burden + very common in JS open source ecosystem so we can get some help with that.  Also, windows support.

Reply:
maybe? but considering it's a monorepo and husky wasn't having fun with that before, I figured we could just follow the same approach

Reply:
Yeah, that's because we were running multiple different pre-commit scripts, here's there's really just one.  Its easy enough to do later.